cmd/object-handlers.go

```diff
@@ -178,6 +178,10 @@ func (api objectAPIHandlers) SelectObjectContentHandler(w http.ResponseWriter, r
 		writeErrorResponse(w, ErrInvalidQuoteFields, r.URL)
 		return
 	}
+	if len(selectReq.InputSerialization.CSV.RecordDelimiter) > 2 {
```

Comment:
Should we also check for unsupported delimiters ? Can you check S3 behavior on this ?

Reply:
ok well here in the s3 docs (https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectSELECTContent.html) i can see the delimiter limitiations are scoped to
 - 2 characters max
 - and the characters can have octals also

So i guess len check is sufficient . What say ?